.env.example

```diff
@@ -13,6 +13,22 @@ CACHE_DRIVER=file
 SESSION_DRIVER=file
 QUEUE_DRIVER=database
 
+GITHUB_KEY=xxx
```

Comment:
please don't use "xxx".
I don't recommend adding these in .env.example. we should introduce how to add them in the documentation.